tests/test_io.py

```diff
@@ -1954,14 +1954,32 @@ def test_mixes_get_variable_s_data(self):
         assert len(full_traj.x) == (ncp + 1)
         assert len(partial_2[vars[0]].x) == (stop_index - start_index)
 
+    def test_cpu_time(self):
+        """ Verify the cumulative CPU time trajectory is never decreasing. """
+        fmu = Dummy_FMUModelME2([], os.path.join(file_path, "files", "FMUs", "XML", "ME2.0", "bouncingBall.fmu"), _connect_dll=False)
+        opts = fmu.simulate_options()
+        opts['dynamic_diagnostics'] = True
+        opts["result_file_name"] = "TestCPUTime.mat"
+        res = fmu.simulate(options = opts)
+
+        rdb = ResultDymolaBinary(opts["result_file_name"])
+        cpu_time = rdb.get_variable_data(f"{DIAGNOSTICS_PREFIX}cpu_time").x
```

Comment:
Does this also affect `get_variables_data`? If yes, worthwhile to also test that one?

Reply:
See https://github.com/modelon-community/PyFMI/pull/291/commits/3d1db1f0f7e2757920baefbbfa9b5bde93732cc9